scripts/gitlab/build_and_test.sh

```diff
@@ -111,6 +112,14 @@ then
     # Map CPU core allocations
     declare -A core_counts=(["lassen"]=40 ["ruby"]=28 ["corona"]=32)
 
+    # If using Multi-project, set up the submodule
+    if [[ -n ${raja_version} ]]
+    then
```

Comment:
I think you could add a message in here like `echo "Updating RAJA submodule to develop"` just so we can easily see in the CI logs it was doing this.